external/loaders/loaders/batches/_batch.py

```diff
@@ -192,10 +200,17 @@ def batches_from_mapper(
     transforms = [_get_batch(data_mapping)]
 
     if needs_grid:
-        transforms += [
-            add_grid_info(res),
-            add_wind_rotation_info(res),
-        ]
+        if gsrm_name == "fv3":
+            transforms += [
+                add_grid_info(res),
+                add_wind_rotation_info(res),
+            ]
+        elif gsrm_name == "scream":
+            transforms += [
+                add_grid_info(res),
+            ]
+        else:
+            raise ValueError(f"gsrm_name {gsrm_name} not supported")
```

Comment:
```suggestion
        transforms.append(add_grid_info(res))
        if vcm.model_name_from_resolution_string(res) == "fv3":
            transforms.append(add_wind_rotation_info(res)])
```

Reply:
Could do something like this (and define appropriate function in `vcm`). It seems like we want to do this `res` -> model name transformation in various places, so we should probably make a single function to do it, and then use that function elsewhere.